exercises/markdown/canonical-data.json

```diff
@@ -120,6 +120,51 @@
             "markdown": "# Start a list\n* Item 1\n* Item 2\nEnd a list"
         },
         "expected": "<h1>Start a list</h1><ul><li>Item 1</li><li>Item 2</li></ul><p>End a list</p>"
+    },
+    {
+        "uuid": "6dca5d10-5c22-4e2a-ac2b-bd6f21e61939",
+        "description": "h7 should not be parsed as it does not exist",
+        "property": "parse",
+        "input": {
+            "markdown": "####### header depth 7 does not exist"
+        },
+        "expected": "####### header depth 7 does not exist"
+    },
+    {
+        "uuid": "9df3f500-0622-4696-81a7-d5babd9b5f49",
+        "description": "with h3 header level",
+        "property": "parse",
+        "input": {
+            "markdown": "### This will be an h3"
+        },
+        "expected": "<h3>This will be an h3</h3>"
+    },
```

Comment:
Could we add the test cases in order of heading levels? What I mean is, the existing test d0f7a31f-6935-44ac-8a9a-1e8ab16af77f checks for h2. I would like to see the test for h3 directly after that, then h4, h5, then the existing test case 13b5f410-33f5-44f0-a6a7-cfd4ab74b5d5 for h6, and then after that the new test case for h7.

So I am asking for those new test cases to be reordered and moved up, near the existing test cases for h2 and h6 🙂

Reply:
Great idea! Re-ordered them to have them all together. Thanks for the review!